sanic/ws.py

```diff
@@ -0,0 +1,53 @@
+from sanic.server import HttpProtocol
+from httptools import HttpParserUpgrade
+from websockets import handshake, WebSocketCommonProtocol
+from websockets import ConnectionClosed  # noqa
+
+
+class WebSocketProtocol(HttpProtocol):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.ws = None
```

Comment:
I would say `ws` in this instance too should be renamed to `websocket` or `web_socket`